resources/definitions/kosher.def.json

```diff
@@ -0,0 +1,61 @@
+{
+    "version": 2,
+    "name": "Kosher",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Venkat Kamesh",
+        "manufacturer": "Sri Vignan Technologies",
+        "weight": 3,
+        "file_formats": "text/x-gcode",
+        "platform": "ultimaker3_platform.obj",
+        "platform_texture": "koshertext.png",
+        "platform_offset": [0, 0, 0],
+        "has_material": true,
+        "has_variants": true,
+        "preferred_variant_name": "0.4 mm",
+        "machine_extruder_trains":
+        {
+            "0": "kosher_extruder_0",
+            "1": "kosher_extruder_1"
+        }
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "Kosher" },
+        "machine_width": { "default_value": 220 },
+        "machine_height": { "default_value": 300 },
+        "machine_depth": { "default_value": 220 },
+        "machine_heated_bed": { "default_value": true },
+        "machine_acceleration": {"default_value": 3000},
+        "machine_max_feedrate_x": { "value": 300 },
+        "machine_max_feedrate_y": { "value": 300 },
+        "machine_max_feedrate_z": { "value": 40 },
+        "machine_max_feedrate_e": { "value": 150 },
+        "machine_use_extruder_offset_to_offset_coords": {"default_value": false},
+        "machine_center_is_zero": {"default_value": false},
+        "gantry_height": {"value": "48"},
+        "line_width": {"value": "machine_nozzle_size"},
+        "speed_travel": {"maximum_value": "300", "value": "200"},
+        "optimize_wall_printing_order": { "value": "True" },
+        "material_diameter": { "default_value": 1.75},
+        "retraction_amount": {"default_value": 6.5},
+        "retraction_speed": { "default_value": 30},
+        
+		"adhesion_type": { "default_value": "skirt" },
+        "machine_gcode_flavor": { "default_value": "Marlin"},
+	    "ironing_enabled":{"label": "Enable Ironing",
+                    "description": "Go over the top surface one additional time, but this time extruding very little material. This is meant to melt the plastic on top further, creating a smoother surface. The pressure in the nozzle chamber is kept high so that the creases in the surface are filled with material.",
```

Comment:
Why override the description with the same value?

Reply:
Thank you 
We will update it